src/pages/Comments/index.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Comment from '../../components/Comment';
+
+function structureComments(comments, showReplies, replyDisplayState, slug, postReplyToComment) {
+  const jsxComments = [];
+  for (let i = 0; i < comments.length; i += 1) {
+    const comment = comments[i];
+    jsxComments.push(<Comment key={comment.id} commentObject={comment} showReplies={showReplies} replyDisplayState={replyDisplayState} slug={slug} postReplyToComment={postReplyToComment} />);
+  }
+  return jsxComments;
+}
+
+const Comments = ({ comments, showReplies, replyDisplayState, slug, postReplyToComment }) => (
+  <div>
+    <div style={{ marginTop: 70, marginBottom: 15 }}>
```

Comment:
@ezrogha put these in css file

Reply:
Thanks for that feedback @dannylwe,
It has been implemented